.github/workflows/local.yml

```diff
@@ -55,8 +55,21 @@ jobs:
       uses: mxschmitt/action-tmate@v2
 
     - name: Build GraphScope
+      env:
+        OS : ${{ matrix.os }}
       run: |
-        /bin/bash ${GITHUB_WORKSPACE}/scripts/deploy_local.sh build_and_deploy --verbose --build_type debug
+        pushd ${GITHUB_WORKSPACE}
+        source ${HOME}/.graphscope_env
+        if [ "${OS}" == "macos-10.15" ];
+        then
+            # need libomp.dylib to find MPI_CXX
+            export LDFLAGS=-L/usr/local/opt/llvm/lib
+            make graphscope WITH_LEARNING_ENGINE=OFF NETWORKX=OFF BUILD_TYPE=debug
+        else
+            export WITH_LEARNING_ENGINE=ON
+            make graphscope NETWORKX=OFF BUILD_TYPE=debug
```

Comment:
why networkx OFF? is that the default value?

Reply:
we don't test networkx in local ci. so we set networkx OFF to save the time of bulding GraphScope.